python/pyspark/mllib/feature.py

```diff
@@ -84,11 +84,14 @@ class Normalizer(VectorTransformer):
     >>> nor2 = Normalizer(float("inf"))
     >>> nor2.transform(v)
     DenseVector([0.0, 0.5, 1.0])
+
+    .. versionadded:: 1.2.0
```

Comment:
Since we won't introduce new API in the micro releases, I think it's better to use `1.2` for short. cc @mengxr